<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/Images/BGP.png);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="minimumSize">
             <size>
              <width>191</width>
              <height>161</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">image: url(:/Images/icon.png);</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QPushButton" name="singleButton">
              <property name="text">
               <string>单人模式</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="doubleButton">
              <property name="text">
               <string>双人模式</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="loadButton">
              <property name="text">
               <string>载入游戏</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="exitButton">
              <property name="text">
               <string>退出游戏</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>菜单</string>
    </property>
    <widget class="QMenu" name="menu_3">
     <property name="title">
      <string>新游戏</string>
     </property>
     <addaction name="action_singlegame"/>
     <addaction name="action_doublegame"/>
    </widget>
    <addaction name="menu_3"/>
    <addaction name="action_load"/>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>关于</string>
    </property>
    <addaction name="action_playmode"/>
    <addaction name="action_item"/>
    <addaction name="action_about"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_playmode">
   <property name="text">
    <string>游戏玩法</string>
   </property>
  </action>
  <action name="action_item">
   <property name="text">
    <string>道具说明</string>
   </property>
  </action>
  <action name="action_about">
   <property name="text">
    <string>关于我们</string>
   </property>
  </action>
  <action name="action_load">
   <property name="text">
    <string>载入游戏</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="text">
    <string>退出游戏</string>
   </property>
  </action>
  <action name="action_singlegame">
   <property name="text">
    <string>单人游戏</string>
   </property>
  </action>
  <action name="action_doublegame">
   <property name="text">
    <string>双人游戏</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
